modal/_partial_function.py

```diff
@@ -36,6 +36,7 @@ class _PartialFunctionFlags(enum.IntFlag):
     EXIT = 16
     BATCHED = 32
     CLUSTERED = 64  # Experimental: Clustered functions
+    CONCURRENT = 128
```

Comment:
Do we need this flag if we have the max_concurrent_input attribute on the _PartialFunction anyway? This might apply to other flags here too though...
I think the purpose of the flags was be able to look up sets of functions that need special handling at different lifecycle stages (or deploy time), which shouldn't be needed for concurrent?

Reply:
Hm perhaps not.

Reply:
Actually actually, maybe not since `max_inputs` must be set if you use `@modal.concurrent`.